addons/crm/models/crm_lead.py

```diff
@@ -1912,8 +1912,7 @@ def _pls_get_won_lost_total_count(self, team_results):
     def _pls_prepare_frequencies(self, lead_values, leads_pls_fields, target_state=None):
         """new state is used when getting frequencies for leads that are changing to lost or won.
         Stays none if we are checking frequencies for leads already won or lost."""
-        # Frequencies must include tag_id
-        pls_fields = set(leads_pls_fields + ['tag_id'])
+        pls_fields = set(leads_pls_fields)
```

Comment:
set is not mandatory here (was used to ensure tag_id was not twice in the list after adding it to pls_fields)